File: 01-searching/csharp/LinearSearch.cs
using System;

class LinearSearch
{
   public static int Search(int[] arr, int target)
    {
        for (int i =0; i < arr.Length; i++)
        {
            if (arr[i] == target)
            {
                return i;
            }
        }
            return -1;
    }

    static void Main(string[] args)
    {
        int[] arr = { 1, 3, 5, 7, 9 };
        int target = 5;
        int result = Search(arr, target);

        if (result != -1)
        {
            Console.WriteLine($"Element found at index: {result}");
        }
        else
        {
            Console.WriteLine("Element not found in the array.");
        }
    }
}

File: 01-searching/python/linear_search.py
def linear_search(numbers, target):
    for i, number in enumerate(numbers):
        if number == target:
            return i

    return -1


def main():
    numbers = [10, 25, 30, 45, 50]

    print(linear_search(numbers, 45))
    print(linear_search(numbers, 100))

if __name__ == "__main__":
    main()
